traitsui/examples/demo/Advanced/Multi_thread_demo_2.py

```diff
@@ -65,7 +65,7 @@ class ThreadDemo(HasTraits):
     )
 
     def __init__(self, **traits):
-        super(HasTraits, self).__init__(**traits)
```

Comment:
this should have been `ThreadDemo` and included in #1583 but i missed it earlier.